esrally/driver/driver.py

```diff
@@ -661,6 +661,16 @@ def retrieve_cluster_info(self, es):
             self.logger.exception("Could not retrieve cluster info on benchmark start")
             return None
 
+    def retrieve_build_hash_from_nodes_info(self, es):
+        try:
+            nodes_info = es["default"].nodes.info(filter_path="**.build_hash")
+            nodes = list(nodes_info["nodes"].keys())
```

Comment:
What do you think about getting the build hash this way instead of populating a list?

```python
In [52]: next(iter(nodes_info["nodes"].items()))[1]["build_hash"]
Out[52]: '4890e5b6b19f9a20c5f62318cf600be156fbc5e3'
```

Reply:
I find that much harder to read than the current implementation.